assets/shell-quote.js

```diff
@@ -25,6 +25,7 @@ var META = '|&;()<> \\t';
 var BAREWORD = '(\\\\[\'"' + META + ']|[^\\s\'"' + META + '])+';
 var SINGLE_QUOTE = '"((\\\\"|[^"])*?)"';
 var DOUBLE_QUOTE = '\'((\\\\\'|[^\'])*?)\'';
+var HTTP_URL_MATCHER  = "http[^ ]+";
```

Comment:
@SahilChoudhary22 can we check the shell-quote npm module, if there are any open issues around this or if there is any handling already for this?